src/scenes/Office/Ppm/StoragePanel.js

```diff
@@ -83,7 +94,7 @@ function mapStateToProps(state, props) {
       total_sit_cost: formatCents(ppm.total_sit_cost),
       days_in_storage: ppm.days_in_storage,
     },
-
+    storageExpenses: filter(moveDocuments, ['moving_expense_type', 'STORAGE']),
```

Comment:
(Nit pick) Since moveDocuments isn't used, you can delete line 88 in favor of the below suggestion.
```suggestion
    storageExpenses: filter(props.moveDocuments, ['moving_expense_type', 'STORAGE']),
```